src/types/aws/apes/donation.ts

```diff
@@ -57,10 +57,9 @@ export type DonationsQueryParams = {
   beforeDate?: string;
   chainName?: string;
   denomination?: string;
-  status?: string;
+  status?: "PENDING" | "RECEIVED";
```

Comment:
Nit: Consider verbiage that is closer to our AWS/DB terminology for statuses?
```suggestion
  status?: "ONHOLD" | "FINALIZED";
```

Reply:
We show these same terms to users:
- [src/pages/Donations/MobileTable.tsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9.2/src/pages/Donations/MobileTable.tsx#L76)
- [src/pages/Donations/Table.tsx](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9.2/src/pages/Donations/Table.tsx#L137)
- [src/pages/Donations/Filter/constants.ts](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9.2/src/pages/Donations/Filter/constants.ts#L137)

Do we change that as well?

Reply:
Ah, nm then. We can leave as is. Thanks! :)

Reply:
Reverted.